apis/v1alpha1/rhmi_types.go

```diff
@@ -128,6 +131,7 @@ var (
 	OperatorVersionMonitoringSpec      OperatorVersion = "1.0"
 	OperatorVersionMarin3r             OperatorVersion = "0.7.0"
 	OperatorVersionGrafana             OperatorVersion = "3.10.2"
+	OperatorVersionObservability       OperatorVersion = "3.0.1"
```

Comment:
latest version of the OO is 3.0.4 may be worth changing to this 
It will give us access to more label selectors
https://github.com/bf2fc6cc711aee1a0c2a/observability-operator/blob/main/api/v1/observability_types.go#L61-L78

Reply:
Think we will also need it for https://issues.redhat.com/browse/MGDAPI-1879, as the Alertmanager config is not included in 3.0.1. https://github.com/bf2fc6cc711aee1a0c2a/observability-operator/blob/main/api/v1/observability_types.go#L77

Reply:
done